### PR TITLE
fix "Alt + keyboard shortcuts do not work"

### LINKED ISSUE #6994: Menu View: Alt+ keyboard shortcuts do not work.
JabRef 5.2--2020-10-07--4b1b7b4
Linux 4.9.0-13-amd64 amd64 
Java 15

- [X] **Mandatory**: I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

The View menu lists the following shortcuts (among others):
- 'Alt+3' for Groups interface
- 'Alt+0' for OpenOffice/Libreoffice
- 'Alt+4' for Web search

None of them are working.
I am using the default key bindings (and, to make sure, I reset them in the Key binding window).

So, the issue is more general than described in #6646

### PR DESCRIPTION
**Pull request by Fu188**
User can use CheckMenuItem shortcut only when they click the tab. That is because the event will not be created until user click the menu in original version. Such that, I create the event when we intial the GUI. In this case, the issue is fixed. Fixes #6994

- [x] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Review by Siedlerchr — APPROVED** [review]
Thanks for your contribution! Looks good to me.

**Review by calixtus — APPROVED** [review]
The reason for the just-in-time creation of the menu is not clear to me, it was introduced in #4707 by @davidemdot , but I have found nothing in the discussion about it. Think it's okay to remove it.

File: src/main/java/org/jabref/gui/JabRefFrame.java [hunk 1]
```diff
@@ -830,25 +830,23 @@ private MenuBar createMenu() {
         SidePaneComponent openOffice = sidePaneManager.getComponent(SidePaneType.OPEN_OFFICE);
 
         view.getItems().add(new SeparatorMenuItem());
-        view.setOnShowing(event -> {
-            view.getItems().clear();
-            view.getItems().addAll(
-                    factory.createCheckMenuItem(webSearch.getToggleAction(), webSearch.getToggleCommand(), sidePaneManager.isComponentVisible(SidePaneType.WEB_SEARCH)),
-                    factory.createCheckMenuItem(groups.getToggleAction(), groups.getToggleCommand(), sidePaneManager.isComponentVisible(SidePaneType.GROUPS)),
-                    factory.createCheckMenuItem(openOffice.getToggleAction(), openOffice.getToggleCommand(), sidePaneManager.isComponentVisible(SidePaneType.OPEN_OFFICE)),
+        view.getItems().clear();
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/JabRefFrame.java:833]:
```
view.getItems().add(new SeparatorMenuItem());
view.getItems().clear();
```
This seems to be totally superfluous now.

**Review by calixtus — CHANGES_REQUESTED** [review]
Hi @Fu188 , I tried to fix the remaining issue I mentioned quickly by myself, but it seems, that you did not grant write access to your PR branch. So please fix the remaining issue, then we can merge.
Also Changelog.md needs some conflict resolution.

**Comment by Fu188** [comment]
> Hi @Fu188 , I tried to fix the remaining issue I mentioned quickly by myself, but it seems, that you did not grant write access to your PR branch. So please fix the remaining issue, then we can merge.
> Also Changelog.md needs some conflict resolution.

I have fix the remaining issue. Thanks for your merge!

**Comment by calixtus** [comment]
Now you have introduced a lot of indentation changes in JabRefFrame (https://github.com/JabRef/jabref/pull/7379/files). Please undo them.

**Comment by Fu188** [comment]
> Now you have introduced a lot of indentation changes in JabRefFrame (https://github.com/JabRef/jabref/pull/7379/files). Please undo them.

I have undo them and it's ok now.

**Review by calixtus — APPROVED** [review]
Looks good to me. Thanks!

**Review by tobiasdiez — APPROVED** [review]
LGTM! Thanks

**Pull request merged**